clients/lsp-netbeans.el

```diff
@@ -0,0 +1,129 @@
+;;; lsp-netbeans.el --- Apache Netbeans Language Server Client settings         -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2021  Tim Felgentreff
+
+;; Author: Tim Felgentreff <timfelgentreff@gmail.com>
+;; Keywords: java groovy graalvm lsp
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+;;; Commentary:
+
+;; LSP and DAP client for the Apache Netbeans LSP server.
+
+;; Currently missing many features that are in the VSCode extension, including, but not limited to:
+;; - All custom commands, such as
+;;     java.build.workspace
+;;     java.generate.getters
+;;     java.generate.setters
+;;     java.generate.getters.setters
+;;     java.generate.implementMethod
+;;     java.generate.overrideMethod
+;;     java.generate.equals.hashCode
+;;     java.generate.delegateMethod
+;;     java.generate.constructor
+;;     java.generate.logger
+;;     java.generate.toString
+;;     graalvm.pause.script
+;;     java.super.implementation
+;; - Groovy support
+;; - Managing/running/debugging unittest directly
+;; - GraalVM Native Image launch/debug configuration
+;; - Attach to JVM process debug configs
+;; - Launch JVM debug config
+
+;;; Code:
+
+(require 'lsp-mode)
+(require 'dap-mode)
```

Comment:
The CI is failing because dap-mode is not a dependency of lsp-mode, you need to check if dap-mode is installed or move this client to a separated repo which I don't think it makes sense right now

Reply:
Alternatively, can  you extract the dap part in dap-mode repo?